content/en/continuous_integration/guides/pull_request_comments.md

```diff
@@ -17,24 +17,26 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-Datadog integrates with GitHub to show test result summaries and error messages for failed tests in pull request comments.
+If you're using the Test Visibility product, Datadog can integrate with GitHub to show test results summaries directly
+in your pull requests. The summary contains an overview of the tests executions, flakiness information and
+error messages for failed tests.
+With this report, the developers get instant feedback about the results of their tests, and they are able to
+debug any failed/flaky tests without leaving the pull requests view.
 
 {{< img src="ci/github_comments_light.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}
 
 ### Compatibility
 
 This integration is only available for test services hosted on `github.com`.
 
-## Install the GitHub integration
+## Enable test summaries
 
-To install the [GitHub integration][1]:
+Enabling test summaries is very quick and can be done in two simple steps. The first step is to install the [GitHub integration][1]:
```

Comment:
```suggestion
You can enable test summaries in GitHub by following the next steps:

1. Install the [GitHub integration][1]:
```

Reply:
This is how I wrote that in principle, but then you have nested ordered lists. Something like:

1. Install Github integration:
    a. Navigate to the Configuration tab...
    b. Give the application...
    
2. Enable test summaries for a test service...

Which is something that I didn't really see in our UI in other places. Do you think that would be better?

Reply:
Yes, I'd say so IMO. I could locate an example for Jenkins (unordered though) https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/pipelines/jenkins.md?plain=1#L794-L805

Reply:
👍 Changed to be an ordered list!